src/backend/distributed/shared_library_init.c

```diff
@@ -1564,6 +1662,20 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
+	DefineCustomIntVariable(
+		"citus.max_high_priority_background_processes",
+		gettext_noop("Sets the maximum number of background processes "
+					 "that can have their CPU priority increased at the same "
+					 "time on a specific node."),
```

Comment:
my expectation of this setting is that it is global, though it's specific to a particular move.

Reply:
At the moment that is equivalent to being global, since there's only one move/split happening at the same time. When we have parallel splits we would need some shared memory counter, similar to how citus.shared_pool_size works.